NBitcoin/Crypto/Cryptsharp/Pbkdf2.cs

```diff
@@ -138,7 +138,7 @@ public byte[] Read(int count)
 		/// <param name="derivedKeyLength">The desired length of the derived key.</param>
 		/// <returns>The derived key.</returns>
 #if USEBC || WINDOWS_UWP || NETCORE
-		public static byte[] ComputeDerivedKey(IMac hmacAlgorithm, byte[] salt, int iterations,
+		internal static byte[] ComputeDerivedKey(IMac hmacAlgorithm, byte[] salt, int iterations,

```

Comment:
This change makes the class useful for your project but it is not usable for others using NETCORE or UWP.

It is clear that making `ComputeDerivedKey` comes with problems. However, if I am working on a netcore app, i will have the `Pbkdf2` class available but without one of its main methods to invoke! After all, `pbkdf` means password based key derivation function, and `ComputeDerivedKey` is all what this class is about.